src/widget.rs

```diff
@@ -120,7 +120,25 @@ impl<'a> Widget for Renderer<'a> {
         let top_col = next_scroll_top(top_col, cursor.1 as u16, width);
 
         let (text, style) = if !self.0.placeholder.is_empty() && self.0.is_empty() {
+            #[cfg(any(
+                feature = "tuirs-crossterm",
+                feature = "tuirs-termion",
+                feature = "tuirs-no-backend",
+            ))]
```

Comment:
In tui-rs, `Text` does not have `push_span` method, so I did not support them.

When I did not put above `cfg`, I got this error.
```shell
error[E0599]: no method named `push_span` found for struct `tui::text::Text` in the current scope
   --> src/widget.rs:[12](https://github.com/kyu08/tui-textarea/actions/runs/10096172842/job/27918026333#step:18:13)9:22
    |
129 |                 text.push_span(self.0.placeholder.as_str());
    |                      ^^^^^^^^^ method not found in `Text<'_>`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `tui-textarea` (lib) due to 1 previous error
warning: build failed, waiting for other jobs to finish...
error: could not compile `tui-textarea` (lib test) due to 1 previous error
```